.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,72 @@
+name: CI/CD Pipeline
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    if: ${{ !contains(github.event.head_commit.message, 'wip') && !contains(github.event.head_commit.message, 'docs') }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 'current'
+
+      - name: Configure Git
+        run: git config --global url."https://git-push-pull:$GL_TOKEN@gitlab.reclaimprotocol.org".insteadOf "https://gitlab.reclaimprotocol.org"
+        env:
+          GL_TOKEN: ${{ secrets.GL_TOKEN }}
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Run tests
+        run: npm run test
+
+  deploy:
+    runs-on: ubuntu-latest
+    needs: test
+    if: ${{ !contains(github.event.head_commit.message, 'wip') && !contains(github.event.head_commit.message, 'docs') }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up AWS CLI
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install zip -y
+          sudo apt-get install awscli -y
+
+      - name: Prepare environment
+        run: echo -e "$ENV" > .env.production
+
+      - name: Replace GL_TOKEN in docker-compose
```

Comment:
this step is no longer required please remove

Reply:
ok